repository/mongo/repository.go

```diff
@@ -0,0 +1,92 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type Repository struct {
+	Db *mongo.Database
+}
+
+func NewMongoDb(ctx context.Context, db *mongo.Database) (*mongo.Database, error) {
+	usersCollection := db.Collection("users")
```

Comment:
why don't use already created methods as `usePostsCollection` etc and keep it in one place?